frontend/src/components/PdfPreview.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { Document, Page } from 'react-pdf/dist/entry.noworker';
+import { Paper, CircularProgress, Button, MobileStepper, withStyles } from 'material-ui';
+import KeyboardArrowLeft from 'material-ui-icons/KeyboardArrowLeft';
+import KeyboardArrowRight from 'material-ui-icons/KeyboardArrowRight';
+
+const styles = theme => ({
+  preview: {
+    marginTop: theme.spacing.unit * 2
+  },
+  progress: {
+    padding: theme.spacing.unit * 4,
+    textAlign: 'center'
+  },
+  page: {
+    display: 'flex',
+    justifyContent: 'center'
+  },
+  stepper: {
+    padding: 0,
+    paddingBottom: theme.spacing.unit
+  }
+});
+
+export class PdfPreview extends Component {
+  state = {
+    total: 0,
```

Comment:
Peut-être pas clair que ce sont des pages... Mais à voir

Reply:
but wait, are you speaking french now ?

Reply:
by the way, i won't fix this